### ISSUE TITLE
Comma mis-indented in records with multiple constructors

### ISSUE THREAD

**Issue by koterpillar** [commercialhaskell/hindent#409]
```haskell
data Foo = FooAlpha { one :: Int, four :: Bool }
         | FooBeta { one :: Int, three :: String }
```

becomes (see commas):

```haskell
data Foo
    = FooAlpha { one :: Int
              ,  four :: Bool}
    | FooBeta { one :: Int
             ,  three :: String}
```

Removing `FooBeta` gives a better result:

```haskell
data Foo = FooAlpha
    { one :: Int
    , four :: Bool
    }
```

**Comment by sighingnow** [comment]
Hi @koterpillar , which version of hindent do you use when encounter this problem ? When trying with the hindent built from master branch, I get the following result:

~~~haskell
data Foo
  = FooAlpha { one :: Int
             , four :: Bool }
  | FooBeta { one :: Int
            , three :: String }
~~~

**Comment by koterpillar** [comment]
I've used 5.2.2, confirming that the behavior is fixed in 5.2.3.
